tests/chainer_tests/functions_tests/connection_tests/test_convolution_nd.py

```diff
@@ -231,4 +231,39 @@ def test_call_cudnn_backward(self):
             y.backward()
             self.assertEqual(func.called, self.expect)
 
+
+class TestConvolutionNDNdarraySupplied(unittest.TestCase):
```

Comment:
To improve readability, consider renaming to `TestConvolutionNDarraySupplied` (without the second **nd**)